package.json

```diff
@@ -15,11 +15,11 @@
     "create:sub": "npm run typeorm -- subscriber:create -d src/subscribers -n"
   },
   "_moduleAliases": {
-    "@types": "dist/src/types",
-    "@controllers": "dist/src/controllers",
-    "@entities": "dist/src/entities",
-    "@interfaces": "dist/src/interfaces",
-    "@services": "dist/src/services"
+    "@types": "src/types",
+    "@controllers": "src/controllers",
+    "@entities": "src/entities",
+    "@interfaces": "src/interfaces",
+    "@services": "src/services"
```

Comment:
I thought I fixed this in https://github.com/MohammedAl-Rowad/Erela/pull/6

but nope